client/components/sticky-panel/style.scss

```diff
@@ -1,7 +1,28 @@
+.sticky-panel {
```

Comment:
Is it okay to modify this Calypso component directly? Has anyone checked to see the impact to the rest of Calypso? We probably need a checklist of all the places it's used so we can check them before merging.